resources/options.html

```diff
@@ -156,6 +156,14 @@
     </div>
 
     </div>
+
+    <button class="btn waves-effect lime waves-light" type="submit" id="restore">
+        <div>
+            <a style="font-family: Roboto,sans-serif; font-size:11pt; color:white">Restore Defaults</a>
```

Comment:
Oeh, I didn't know that. Must have been added on friday while I was in hospital. :P 
I just noticed it, thanks! :D